tests_integration_with_local_chain/markets/omen/test_stealing_on_markets.py

```
@@ -36,15 +36,15 @@
 def test_stealing_on_markets(
-    accounts: list[TestAccount],
+    eoa_accounts: list[TestAccount],
     local_web3: Web3,
 ) -> None:
     """
     TLDR: If someone places incorrect bet and the liquidity is removed, that someone is locked-in and liquidity provider will receive it.
     """
 
     # Get two accounts, one will create a job market (A) and one will try to sabotage it (B)
-    account_A, account_B = accounts[2], accounts[3]
+    account_A, account_B = eoa_accounts[2], eoa_accounts[3]
```

Pylint (3.3.7)

[convention] Variable name "account_A" doesn't conform to snake_case naming style

(C0103)